WrightTools/artists.py

```diff
@@ -61,6 +61,9 @@ def _parse_cmap(self, data=None, channel_index=None, **kwargs):
             if isinstance(kwargs['cmap'], string_type):
                 kwargs['cmap'] = colormaps[kwargs['cmap']]
             return kwargs
+        elif 'signed' in kwargs.keys():
+            kwargs['cmap'] = colormaps['signed']
+            return kwargs
```

Comment:
This can be accomplished with `cmap='signed'`
I'm hesitant to add behavior that is solely dependent on the presence of a kwarg, I feel like this could be confusing, e.g. `signed = False` here, would result in a signed colormap being used. If we wish to treat signed as a 'special' kwarg, I think we should look at the value.

Reply:
Agreed, but more fundamental issues with `signed` kwarg have been raised.  Decided to remove the kwarg altogether (346b527f620f45109ccd3d0035e016ecdfbf0f06)